### ISSUE TITLE
Pool creation fails with "cannot label PHYSICALDRIVEX: unable to read disk capacity"

### ISSUE THREAD

**Issue by jpovixwm** [openzfsonwindows/ZFSin#155]
Hi.
I received a new HDD today, and wanted to create a ZFS pool on it using ZFSin. I'm new to ZFS in general, so I just followed the readme, and installed the 0.19 (Systkin) release.
I identified the physical drive with wmic, which returned PHYSICALDRIVE1, so I tried to create the first pool, but I could not proceed any further because of an error:
```
C:\Windows\system32>zpool create tank PHYSICALDRIVE1
Expanded path to '\\?\PHYSICALDRIVE1'
DeviceName said OK with '*'
DriveGeometry said OK
LayoutInfo said OK
DiskExtents said NG
cannot label 'PHYSICALDRIVE1': cannot label '\\?\PHYSICALDRIVE1': unable to read disk capacity
```
Am I missing something obvious? I tried running that command on an "uninitialized" drive, on a GPT-initialized one with 100% of unallocated space (no partitions), and also with the 100% of the disk occupied by a single NTFS partition, but the zpool output was exactly the same for all of the mentioned cases.

Meanwhile, I also tried creating a file-based pool, which worked correctly. The issue is only when trying to use the new disk directly. I didn't try it with the drive initialized to MBR, seeing how it limits any partition to 2 TB, which is a limitation I would not accept anyways.

**Comment by lundman** [comment]
That is interesting - ZFS when you give it a wholedisk (like you are) will re-partition the drive with EFI/gpt so even if you did mbr partition it, it will write over it. What size is this disk? Have you found a bug inside libefi maybe which fails to partition it.

Please also try with the latest nightly build from the wiki.

**Comment by jpovixwm** [comment]
The disk is a 4 TB Toshiba HDWE140UZSVA. I can't promise I'll be able to test the nightly build anytime soon because I've already started storing some data on the disk and I have no place to backup to yet.
For what it's worth, I was able to create a ZFS pool on this drive using FreeBSD running as a VirtualBox guest on the Windows host with direct whole disk access passed thru VirtualBox. Same for ZOL running on Debian in a similar VirtualBox VM.

**Comment by lundman** [comment]
Yeah, don't worry about it - I can find a 4TB disk to test on easily, but not sure I can test it before Obon starts though.

**Comment by AaronEban** [comment]
I encountered the same problem when i ran the same command as @jpovixwm in the windows CMD shell. But it is successful when i run it in the Windows PowerShell. Version 0.23.
